index.ts

```diff
@@ -81,7 +81,7 @@ async function startMeeting(params: RNZoomUsStartMeetingParams) {
 }
 
 export default {
-  ...RNZoomUs,
+  ...RNZoomUs as {},
```

Comment:
:thinking: I thought about it and a better way to do it is to specify exactly what will be exported and type it here or not to export `RNZoomUs`. I mean what is the point of exporting `RNZoomUs` if we export particular exposed api below?

Reply:
I thought there were some extra methods. Removing...

Reply:
thnx, if someone was using anything directly from `RNZoomUs` we can add it. but that should be explicit